CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# 🧑‍💻 Contributing to ZenML Projects
+
+A big welcome and thank you for considering contributing to ZenML Projects! It’s people
+like you that make it a reality for users
+in our community.
+
+Reading and following these guidelines will help us make the contribution
+process easy and effective for everyone
+involved. It also communicates that you agree to respect the developers' time
+management and develop these open-source projects. In return, we will reciprocate that respect by reading your
+issue, assessing changes, and helping
+you finalize your pull requests.
+
+## ⚡️ Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+* [Getting Help](#getting-help)
+
+## 🧑‍⚖️ Code of Conduct
+
+We take our open-source community seriously and hold ourselves and other
+contributors to high standards of communication.
+By participating and contributing to this project, you agree to uphold
+our [Code of Conduct](https://github.com/zenml-io/zenml-projects/blob/master/CODE-OF-CONDUCT.md).
+
+## 🛫 Getting Started
+
+Contributions are made to this repo via Issues and Pull Requests (PRs). A few
+general guidelines that cover both:
+
+- To report security vulnerabilities, please get in touch
+  at [support@zenml.io](mailto:support@zenml.io), monitored by
+  our security team.
+- Search for existing Issues and PRs before creating your own.
+- Search on the [public Crowd Dev](https://open.crowd.dev/zenml) page to see 
+if the issue has been addressed  before.
+- We work hard to make sure issues are handled on time, but it could take a
+  while to investigate the root cause depending on the impact.
+
+A friendly ping in the comment thread to the submitter or a contributor can help
+draw attention if your issue is blocking.
+
+The best way to start is to check the
+[`good-first-issue`](https://github.com/zenml-io/zenml-dashboard/labels/good%20first%20issue)
+label on the issue board. The core team creates these issues as necessary
+smaller tasks that you can work on to get deeper into ZenML dashboard internals. These
+should generally require relatively simple changes, probably affecting just one
+or two files which we think are ideal for people new to ZenML dashboard.
+
+The next step after that would be to look at the
+[`good-second-issue`](https://github.com/zenml-io/zenml-dashboard/labels/good%20second%20issue)
+label on the issue board. These are a bit more complex, might involve more
+files, but should still be well-defined and achievable to people relatively new
+to ZenML dashboard.
```

Comment:
Do we need this in here? I'm actually wondering whether we need / want to strip this guide down to something a bit more focused on the projects themselves?